src/mlpack/tests/cf_test.cpp

```diff
@@ -730,12 +728,14 @@ TEST_CASE("RecommendationAccuracyBiasSVDTest", "[CFTest]")
  * Make sure recommendations that are generated are reasonably accurate
  * for SVDPlusPlus method.
  */
-// This test is commented out because it fails and we haven't solved it yet.
-// Please refer to issue #1501 for more info about this test.
-// TEST_CASE("RecommendationAccuracySVDPPTest", "[CFTest]")
-// {
-//   RecommendationAccuracy<SVDPlusPlusPolicy>();
-// }
+TEST_CASE("RecommendationAccuracySVDPPTest", "[CFTest]")
+{
+  math::RandomSeed(25);
```

Comment:
I think this line is just a WIP line, but we shouldn't set a hardcoded seed inside of the test.  Instead our goal is that this decomposition works reliably with different seeds.  In general you should aim for settings of this test such that the failure rate is less than 1 in 1000 (that's what I usually test for).